docs/VR/GetSupportedLanguages/assets/GetSupportedLanguages.txt

```diff
@@ -0,0 +1,44 @@
+participant App
+participant SDL
+participant HMI
+
+HMI->SDL:BC.OnReady
+SDL->HMI:VR.IsReady
+HMI-->SDL:VR.IsReady (code:0)
+note over SDL:<align:center>`HMICapabilitiesCacheFile`\nis not available
+SDL->SDL:Create `HMICapabilitiesCacheFile`
+
+opt Use of cached capabilities data
+
+opt On system start up
+note over SDL:<align:center> Capabilities data is not available\nin`HMICapabilitiesCacheFile`
+SDL->HMI:VR.GetSupportedLanguages
+HMI-->SDL:VR.GetSupportedLanguages (params)
+SDL->SDL:Cache received data\nin `HMICapabilitiesCacheFile`
+App->SDL:GetSupportedLanguages
```

Comment:
There is no `GetSupportedLanguages` RPC on the mobile side

Reply:
fixed in e32e4d6